plugins_tools/aboutmw/OSX/eID Middleware/eID Middleware/Base.lproj/MainMenu.xib

```diff
@@ -57,6 +57,23 @@
                         </items>
                     </menu>
                 </menuItem>
+                <menuItem title="File" id="sQX-hx-PSV">
+                    <modifierMask key="keyEquivalentModifierMask"/>
+                    <menu key="submenu" title="File" id="bQ4-eu-hHJ">
+                        <items>
+                            <menuItem title="Page Setup…" keyEquivalent="P" id="0DE-TF-sO4">
+                                <connections>
+                                    <action selector="runPageLayout:" target="-1" id="dxE-rN-MNq"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Print…" keyEquivalent="p" id="kcM-FJ-Kbh">
+                                <connections>
+                                    <action selector="print:" target="-1" id="RWV-kv-9P9"/>
+                                </connections>
+                            </menuItem>
+                        </items>
+                    </menu>
+                </menuItem>
```

Comment:
Why? Please explain in the commit message what the changes in this file accomplish.